### PR TITLE
Add myself to poule to get 2w/old PRs

### PR DESCRIPTION
**Pull request by fntlnz**

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]
LGTM :tiger:

File: .DEREK.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ curators:
   - rheinwein
   - ripcurld0
   - thajeztah
+  - fntlnz
```
Comment (thaJeztah) [inline .DEREK.yml]: Can you keep the lists alphabetical? 🤗

File: poule.yml [hunk 2]
```diff
@@ -124,5 +124,6 @@
                 "vdemeester",
                 "vieux",
                 "yongtang",
+                "fntlnz",
```
Comment (thaJeztah) [inline poule.yml]: Same for this one :smile:

**Review by thaJeztah — CHANGES_REQUESTED** [review]
one nit, but LGTM otherwise 🤗

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Merging, as this file isn't tested in CI :smile:

**Pull request merged**